packages/flutter_tools/test/general.shard/ios/ios_device_start_prebuilt_test.dart

```diff
@@ -319,7 +319,7 @@ void main() {
             '--disable-service-auth-codes',
             '--observatory-port=60700',
             '--start-paused',
-            '--dart-flags="--foo"',
+            '--dart-flags="--foo,--null_assertions"',
```

Comment:
Is there a test that's just `--null_assertions` without a `--foo`?

Reply:
I added a unit test that tests various flag combinations in computeDartVmFlags